Uni_life_tracker/app/src/main/java/com/generic/plannr/Gateways/UserGateway.java

```diff
@@ -0,0 +1,267 @@
+package com.generic.plannr.Gateways;
+
+import android.annotation.SuppressLint;
+import android.content.ContentValues;
+import android.content.Context;
+import android.database.Cursor;
+import android.database.sqlite.SQLiteDatabase;
+
+import com.generic.plannr.Database.DatabaseClient;
+import com.generic.plannr.Entities.Event;
+import com.generic.plannr.Entities.User;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+
+public class UserGateway implements UserGatewayInterface {
+
+    public SQLiteDatabase db;
+    public DatabaseClient dbclient;
+
+    public UserGateway(Context context) {
+        dbclient = new DatabaseClient(context);
+    }
+
+    /**
+     * Open the database for reading or writing
+     */
+    public void openDatabase() {
+        db = dbclient.getWritableDatabase();
+    }
+
+    /**
+     * Insert user's info into the database
+     *
+     * @param user The User to be inserted
+     */
+    public void saveToDatabase(final User user) {
+        openDatabase();
+        ContentValues cv = new ContentValues();
+        cv.put("NAME", user.getName());
+        cv.put("UNIVERSITY", user.getSchool());
+        cv.put("EMAIL", user.getEmail());
+        cv.put("PASSWORD", user.getPassword());
+        cv.put("LOGGEDIN", 1);
```

Comment:
Maybe add a _ between Logged and in because its technically two works

Reply:
done